build/run-ci.yaml

```diff
@@ -7,9 +7,14 @@
   - name: Run the STF CI system
     import_role:
       name: stf-run-ci
-          
+
+  - name: Create Log directory
+    file:
+      path: build/working/logs
```

Comment:
Please replace this with an absolute path.

I would suggest "{{ playbook_dir }}/working/logs", which is "path/to/STO/build/working/logs"